lua/git-links/utils.lua

```diff
@@ -88,11 +109,13 @@ local function will_fail()
 end
 
 Utils.Steps = {
-  { func = fetch_url,       name = "Fetch Remote URL" },
-  { func = fetch_hash,      name = "Fetch Hash" },
-  { func = set_repo_type,   name = "Detect Repo Type" },
-  { func = fetch_file_info, name = "Fetch File Info" },
-  { func = get_line_number, name = "Find Line Number" },
+  { func = check_branch_remote, name = "Check Branch Exists On Remote" },
+  { func = check_commit,        name = "Verify Commits" },
+  { func = fetch_url,           name = "Fetch Remote URL" },
+  { func = fetch_hash,          name = "Fetch Hash" },
```

Comment:
Would need to move this up in the chain if the hash is checked instead of the branch as noted above.